docs/en/compatibility.md

```diff
@@ -1,3 +1,53 @@
+### v2.0.0rc1
+
+The OpenMMLab team released a new generation of training engine [MMEngine](https://github.com/open-mmlab/mmengine) at the World Artificial Intelligence Conference on September 1, 2022. It is a foundational library for training deep learning models. Compared with MMCV, it provides a universal and powerful runner, an open architecture with a more unified interfaces, and a more customizable training process.
+
+At the same time, MMCV released [2.x](https://github.com/open-mmlab/mmcv/tree/2.x) release candidate version and will release 2.x officially version on January 1, 2023. In version 2.x, it has the following changes:
+
+(1) It removes the following components:
+
+- Removed `mmcv.fileio` module in PR [#2179](https://github.com/open-mmlab/mmcv/pull/2179) and used fileio module from mmengine wherever fileio is required.
+- Removed four modules in PR [#2216](https://github.com/open-mmlab/mmcv/pull/2216), `mmcv.runner`, `mmcv.parallel`, `mmcv. engine` and `mmcv.device`
+- Removed all classes in `mmcv.utils` (eg `Config` and `Registry`) and large Some functions in PR [#2217](https://github.com/open-mmlab/mmcv/pull/2217) , only a few functions related to mmcv are reserved.
+- Removed `mmcv.onnex`, `mmcv.tensorrt` modules and related functions in PR [#2225](https://github.com/open-mmlab/mmcv/pull/2225).
+
+(2) It adds the [`mmcv.transforms`](https://github.com/open-mmlab/mmcv/tree/2.x/mmcv/transforms) data transformation module.
+
+(3) It renamed the package name **mmcv** to **mmcv-lite**, **mmcv-full** renamed to **mmcv** in PR [#2235](https://github.com/open-mmlab/mmcv/pull/2235). Also, change the default value of the environment variable `MMCV_WITH_OPS` from 0 to 1.
```

Comment:
`renamed`, `adds`, `removes`. Should theres tenses should be unified?

Reply:
okay